design/sd-zip-artifacts.md

```diff
@@ -91,7 +91,7 @@ Status(Error):
 
 |Method|URL|Description|
 |:--|:--|:--|
-|POST|/queue/message|enqueue message to unzip file|
+|POST|/queue/message?type=unzip|enqueue message to unzip file|
```

Comment:
[imo]
Maybe we do not need to add parameters here. Parameters are explained in POST DATA section.

Reply:
Thx.
I deleted it.